DigitalLearningSolutions.Web/Views/Frameworks/Developer/AddCompetencyLearningResources.cshtml

```diff
@@ -46,11 +66,25 @@
     <div class="nhsuk-grid-column-three-quarters">
       <div class="search-box-container" id="search">
         @Html.LabelFor(m => m.SearchText, "Search", new { @class = "nhsuk-u-visually-hidden", @for = "search-field" })
-        @Html.TextBoxFor(m => m.SearchText, new { @class = "search-box", id = "search-field", name = "searchString", type = "search", placeholder = "Search", autocomplete = "off" })
-        <button class="nhsuk-button sort-submit" type="submit" asp-route-page="1" asp-route-frameworkId="@Model.FrameworkId" asp-route-frameworkCompetencyId="@Model.FrameworkCompetencyId" asp-route-frameworkCompetencyGroupId="@Model.FrameworkCompetencyGroupId">
+        @Html.TextBoxFor(m => m.SearchText, new { @class = "search-box nhsuk-input", id = "search-field", name = "searchString", type = "search", placeholder = "Search", autocomplete = "off" })
+        <button class="nhsuk-button sort-submit" type="submit"
+                asp-route-page="@Model.Page"
+                asp-route-frameworkId="@Model.FrameworkId"
+                asp-route-frameworkCompetencyId="@Model.FrameworkCompetencyId"
+                asp-route-frameworkCompetencyGroupId="@Model.FrameworkCompetencyGroupId"
+                asp-route-catalogueId="@Model.CatalogueId">
           Search
         </button>
       </div>
+      <div class="filter-value-container">
+        @Html.DropDownListFor(c => c.CatalogueId, dropdownItems,

```

Comment:
Accessibility tests are complaining that this dropdown doesn't have a unique ID. There is a hidden input on the page with the same Id - "CatalogueId"